clang/test/SemaSYCL/buffer_location.cpp

```diff
@@ -14,17 +14,17 @@ struct Base {
   cl::sycl::accessor<char, 1, cl::sycl::access::mode::read,
                      cl::sycl::access::target::global_buffer,
                      cl::sycl::access::placeholder::false_t,
-                     cl::sycl::property_list<
-                         cl::sycl::property::buffer_location<1>>>
+                     cl::sycl::ONEAPI::accessor_property_list<
+                         cl::sycl::INTEL::property::buffer_location::instance<1>>>
       AccField;
 };
 
 struct Captured : Base,
                   cl::sycl::accessor<char, 1, cl::sycl::access::mode::read,
                                      cl::sycl::access::target::global_buffer,
                                      cl::sycl::access::placeholder::false_t,
-                                     cl::sycl::property_list<
-                                         cl::sycl::property::buffer_location<1>>> {
+                                     cl::sycl::ONEAPI::accessor_property_list<
```

Comment:
Formatting looks weird

Reply:
Replaced property with an alias to make lines shorter. Not sure if there's another way to make it look less awkward.